modelling/src/main/java/org/axonframework/modelling/saga/repository/jdbc/SagaSchema.java

```diff
@@ -26,37 +26,53 @@
  */
 public class SagaSchema {
 
-    private static final String DEFAULT_SAGA_ENTRY_TABLE = "SagaEntry";
-    private static final String DEFAULT_ASSOC_VALUE_ENTRY_TABLE = "AssociationValueEntry";
-
     private final String sagaEntryTable;
+    private final String serializedSagaColumn;
```

Comment:
Although I know the `TokenSchema` class does not have a test case, I think it's better to have one for the `SagaSchema` class. Would you mind adding that, @Blackdread?

Reply:
what kind of test to add?

Reply:
I added few tests